### PR TITLE
medical core landing page

### PR DESCRIPTION
**Pull request by esther921**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,231 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,600;0,700;1,400&display=swap" rel="stylesheet">
+  <link href="./styles/index.css" rel="stylesheet">
+</head>
+<body class="main">
+<div class="page1">
```
Comment (Delyc) [inline index.html:10]: Please try to use descriptive and meaningful names (e.g page1 does not describe which block or part on your page)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        </div>
+      </div>
+    </div>
+    </div>
+</div>
+
+
+
+
+<div class="page4">
```
Comment (Delyc) [inline index.html:212]: make use of HTML5 semantics to provide meaning to your HTML page(such as footer, section ...)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+  <div class="page4__body">
+  <h6 class="heading6 heading6--page4">Newsletter</h6>
+  <h3 class="heading3 heading3--page4" >JOIN US</h3>
+  <p  class="paragraph paragraph--page4" >Problems trying to resolve the conflict between the two major realms of classical physics: Newtorian mechanics</p>
+  </div>
+  <div class="page4__footer">
```
Comment (Delyc) [inline index.html:219]: Make sure space left on both sides(left and right) are equal. down below on subscribe input

File: index.html [hunk 4]
```diff
@@ -0,0 +1,231 @@
+<!doctype html>
```
Comment (Delyc) [inline index.html:1]: More general feedback, format your codes as they can be easily readable.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+    </div>
+</div>
+
+
+
+
+<div class="page4">
+
+  <div class="page4__body">
+  <h6 class="heading6 heading6--page4">Newsletter</h6>
```
Comment (mbukeRepo) [inline index.html]: **heading6--page4** what kind of modifier is this

File: index.html [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+
+  </div>
+
+
+
+
+  <div class="page3">
+    <h6 class="heading6 heading6--page3">Team</h6>
+    <h3 class="heading3">Our Team</h3>
+    <p class="paragraph paragraph--page3">Problems trying to resolve the conflict between the two major realms of classical physics: Newtorian mechanics</p>
```
Comment (mbukeRepo) [inline index.html]: **paragraph** is an element not a block. where does **paragraph** element belong to.

**Review by mbukeRepo — CHANGES_REQUESTED** [review]

**Review by Delyc — CHANGES_REQUESTED** [review]

**Review by mbukeRepo — APPROVED** [review]
LGTM

**Review by Delyc — APPROVED** [review]
Nice work!

File: index.html [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+    <div>
```
Comment (Morrism1) [inline index.html]: Having elements without classes is not necessary in BEM convention.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,231 @@
...
+<div class="page4">
+
+  <div class="page4__body">
+  <h6 class="heading6 heading6--page4">Newsletter</h6>
+  <h3 class="heading3 heading3--page4" >JOIN US</h3>
+  <p  class="paragraph paragraph--page4" >Problems trying to resolve the conflict between the two major realms of classical physics: Newtorian mechanics</p>
+  </div>
+  <div class="page4__footer">
+    <input  class="input" type="text" placeholder="Your Email">
+    <button class="input__button"><p class="input__subscribe">Subscribe</p></button>
```
Comment (Morrism1) [inline index.html]: Here we have `input__button` class, I expect this to be inside a BLOCK called `input` but looks it is a sibling with one instead.

Comment (Morrism1) [reply index.html]: Same with `input__subscribe` does not have any BLOCK with `input`

**Review by Morrism1 — CHANGES_REQUESTED** [review]

**Review by Delyc — APPROVED** [review]

File: index.html [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+<div class="page4">
+
+  <div class="page4__body">
+  <h6 class="heading6 heading6--last">Newsletter</h6>
+  <h3 class="heading3 heading3--last" >JOIN US</h3>
+  <p  class="paragraph paragraph--last" >Problems trying to resolve the conflict between the two major realms of classical physics: Newtorian mechanics</p>
+  </div>
+  <div class="page4__footer">
+    <input  class="page4__input" type="text" placeholder="Your Email">
+    <button class="input">
```
Comment (mbukeRepo) [inline index.html:221]: where does **input** element belong to.

**Review by Delyc — APPROVED** [review]

**Review by mbukeRepo — APPROVED** [review]
LGTM

File: index.html (hunk 7 above)
Comment (Morrism1) [reply index.html]: Remember to remove elements with no classes because they are unnecessary.

**Review by Morrism1 — CHANGES_REQUESTED** [review]

File: index.html (hunk 7 above)
Comment (esther921) [reply index.html]: I thought if i want to  be using my block else where i had to make it as a block.

**Pull request merged**